### DISCUSSION TITLE
Allow List Type without predefined values as frontmatter

### DISCUSSION THREAD

**Discussion by pravin-d** [mdelobelle/metadatamenu#446] [Tips and Tricks]
It would be great if this plugin could also support the native list type. Select single/multiple require a predefined value. However there are cases where one would want to just input a list of items. Currently only text is support. No list type.

**Comment by mdelobelle** [comment]
You can use the Yaml type field for it from the beta version https://github.com/mdelobelle/metadatamenu/releases/tag/0.6.0-beta-8
